gerrit_to_github_issues/github_issues.py

```diff
@@ -75,3 +75,52 @@ def get_bot_comment(issue: Issue, bot_name: str, ps_number: str) -> IssueComment
     for i in issue.get_comments():
         if i.user.login == bot_name and ps_number in i.body:
             return i
+
+
+def assign_issues(repo)
+    open_issues = [i for i in repo.get_issues() if i.state == 'open']
+    for issue in open_issues:
+        try_assign(issue)
+
+
+def try_assign(issue):
+    # find the most recent assignment request
+    assignment_request = None
+    for comment in issue.get_comments().reversed:
+        if comment.body == '/assign':
```

Comment:
should this be a contains '/assign' in case someone adds an unnecessary space or extra details like "Hi please /assign this to me."

Reply:
good call. updated